frontend/www/js/omegaup/components/contest/Edit.vue

```diff
@@ -215,7 +215,10 @@
           :data="requests"
           :text-add-participant="T.contestAdduserAddContestant"
           @accept-request="(request) => $emit('accept-request', request)"
-          @deny-request="(request) => $emit('deny-request', request)"
+          @deny-request="
+            (request, resolutionText) =>
+              $emit('deny-request', request, resolutionText)
+          "
```

Comment:
This change could be reverted:
```suggestion
          @deny-request="(request) => $emit('deny-request', request)"
```